tests/release/conftest.py

```diff
@@ -28,7 +32,17 @@
     "wic": generate_women_infants_and_children,
     "tax_1040": generate_taxes_1040,
 }
+DATASET_ARG_TO_FULL_NAME_MAPPER: dict[str, str] = {
+    "acs": "american_community_survey",
+    "cps": "current_population_survey",
+    "census": "decennial_census",
+    "ssa": "social_security",
+    "taxes_1040": "taxes_1040",
+    "taxes_w2_and_1099": "taxes_w2_and_1099",
+    "wic": "women_infants_and_children",
+}
 
+SEED = 0
```

Comment:
This is out of scope for this ticket but I think should definitely be an argument that can be passed in so we can run on different seeds to verify we don't have a bug and aren't just getting lucky. Having it have a default of 0 is perfectly fine.

Reply:
Yeah let's talk about this it should be really easy to implement as well